sdk/dotnet/ContainerService/RegistryWebook.cs

```diff
@@ -11,7 +11,12 @@ namespace Pulumi.Azure.ContainerService
 {
     /// <summary>
     /// Manages an Azure Container Registry Webhook.
+    /// 
+    /// 
+    /// 
+    /// Deprecated: azure.RegistryWebook has been deprecated in favour of azure.RegistryWebhook
     /// </summary>
+    [Obsolete(@"azure.RegistryWebook has been deprecated in favour of azure.RegistryWebhook")]
```

Comment:
I need to try and follow up and make these language specific or something more generic - I can follow up after this is merged - this will at least introduce obsolete for now and we can enhance it :)